ReactAndroid/build.gradle

```diff
@@ -255,8 +247,8 @@ task cleanReactNdkLib(type: Exec) {
 
 task packageReactNdkLibs(dependsOn: buildReactNdkLib, type: Copy) {
     from("$buildDir/react-ndk/all")
-    from("$thirdPartyNdkDir/jsc/jni")
     into("$buildDir/react-ndk/exported")
+    exclude("**/libjsc.so")
```

Comment:
Is this redundant ? By default gradle deps won't bundled in the aar.

Reply:
libjsc.so will be copied into `$buildDir/react-ndk/all` during ndkbuild.
If we don't exclude libjsc.so here, they will be copied to exported folder and packed in AAR.
This exclusion is pretty much same as what we did in old JSC, [e.g. 0.57](https://github.com/facebook/react-native/blob/0.57-stable/ReactAndroid/build.gradle#L239)

Reply:
Can we also exclude libc++_shared.so too so that app packages don't have to do that `pickFirst` thing?

It's a dangerous thing in case there are other dependencies than RN that bring in libc++_shared.so as well.

Reply:
That is a good point and I don't have a good solution right now.
The problem is that jsc-android-buildscripts packed AAR with a C++ runtime (libc++_shared.so).
[Old JSC excludes C++ runtime to AAR. ](https://github.com/facebook/android-jsc/blob/master/install.sh#L15-L18)
That's the difference why old JSC does not have to use pickFirst.

There is also an assumption that NDK maintains ABI compatibilities because RN and JSC may build from different NDK version, e.g. old JSC is built by NDK r10c.
Fortunately, we don't have problems from ABI incompatibilities during these years.

Possible solutions like:

1. Built JSC as static library. This will increase libjsc.so binary size and impact to application APK size.
2. Move jsc-android-buildscripts to RN core and build together. This is increase build time incredibly. (e.g. It takes about 2 hours to build a jsc-android-buildscripts job on CicleCI.)

IMO, excluding libc++_shared.so from jsc-android-buildscripts is better than pickFirst as pickFirst may impact to application level.
But this needs changes from jsc-android-buildscripts side.